src/UI/Factory.php

```diff
@@ -717,4 +717,25 @@ public function legacy($content);
 	 */
 	public function table();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Alerts inform the user about the state of the system. Such as the successful completion, the need for further
+	 *     input  of an actual error or stopping users in their tracks in high-risk tasks.
+	 *   composition: >
+	 *     Alerts are embedded or popup messages comprised of text and optionally buttons.
```

Comment:
When will we use the one or the other?

Reply:
That "popup" stuff IMO is something we really should elaborate and maybe even import directly into this here. My gut feeling ist, that I would always want Confirmation Alert and Failure Alert to be interuptive modals, while info and success should always be embedded.

Reply:
This has been removed from the current proposal.